vue/sbc-common-components/src/main.ts

```diff
@@ -1,12 +1,14 @@
-import Vue from 'vue';
-import App from './App.vue';
-import router from './router';
-import store from './store';
+import Vue from 'vue'
```

Comment:
It's generally best practice to leave semicolons in... unless we are using a styling convention that says otherwise.

Reply:
Eslint keep telling me  those need to be removed.

Reply:
Okay, looks like that may be the vue convention, my bad ;)